packages/opentelemetry-node-tracer/src/NodeTracer.ts

```diff
@@ -14,8 +14,47 @@
  * limitations under the License.
  */
 
-import { BasicTracer, BasicTracerConfig } from '@opentelemetry/basic-tracer';
+import { BasicTracer } from '@opentelemetry/basic-tracer';
 import { AsyncHooksScopeManager } from '@opentelemetry/scope-async-hooks';
+import { ScopeManager } from '@opentelemetry/scope-base';
+import {
+  Attributes,
+  BinaryFormat,
+  HttpTextFormat,
+  Logger,
+  Sampler,
+} from '@opentelemetry/types';
+
+/**
+ * NodeTracerConfig provides an interface for configuring a Node Tracer.
+ */
+export interface NodeTracerConfig {
```

Comment:
`NodeTracerConfig` same as `BasicTracerConfig` except one mandatory attribute. What's the motivation here? I proposed to keep `ScopeManager` optional attribute in https://github.com/open-telemetry/opentelemetry-js/issues/194. If we do that then we don't need to create `NodeTracerConfig` interface.

Reply:
The problem is i cant extend the `BasicTracerConfig` to tell `scopeManager` is optional so i'm forced to make a new interface.
I agree that it should be optional and the `BasicTracer` should create a NoopScopeManager if none is used, if we agree we should open another PR